<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formViewResult</class>
 <widget class="QMainWindow" name="formViewResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modulo Visualização</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>src/principal_icon.png</normaloff>src/principal_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>1</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#037f8c;&quot;&gt;APOSTADORPRO&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#037f8c;&quot;&gt;GERENCIADOR DE APOSTAS ESPORTIVAS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>375</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>375</width>
      <height>85</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 127, 140);
border-radius: 5px;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="rb_matchOdds">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Match Odds</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_DuplaChance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Dupla Chance</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_ambosMarcam">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Ambos Marcam</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_totalGols">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Total de Gols</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_jogosBetano">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Jogos Betano</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_jogosPixbet">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Jogos Pixbet</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_matchJogos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Match Jogos</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tb_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>1251</width>
      <height>400</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <column>
     <property name="text">
      <string>ind</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hora</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeCasa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TimeVisitante</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Odds1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OddsX</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Odds2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MaisGols</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MenosGols</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MarcamS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MarcamN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dupla1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dupla12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dupla2</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_sair">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>597</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_voltar">
    <property name="geometry">
     <rect>
      <x>999</x>
      <y>597</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_simular">
    <property name="geometry">
     <rect>
      <x>859</x>
      <y>597</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style: none;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	border-bottom-color: rgb(4, 35, 38);
	border-right-color:  rgb(4, 35, 38);
}

QPushButton:pressed{
	border-style: insetset;
	border-width: 2px;
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 8px;
	border-top-color: rgb(4, 35, 38);
	border-left-color:  rgb(4, 35, 38);
}</string>
    </property>
    <property name="text">
     <string>Simular Aposta</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>111</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(3, 127, 140);
	border-radius: 12px;
	</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Jogos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
